editors/sc-ide/widgets/main_window.cpp

```diff
@@ -654,6 +654,19 @@ void MainWindow::createMenus()
     menu->addSeparator();
     menu->addAction( mMain->scProcess()->action(ScProcess::ShowQuarks) );
     menu->addSeparator();
+    menu->addAction( mEditors->action(MultiEditor::EvaluateCurrentDocument) );
+    menu->addAction( mEditors->action(MultiEditor::EvaluateRegion) );
+    menu->addAction( mEditors->action(MultiEditor::EvaluateLine) );
+    menu->addAction( mMain->scProcess()->action(ScIDE::ScProcess::StopMain) );
+    menu->addSeparator();
+    menu->addAction( mActions[LookupImplementationForCursor] );
+    menu->addAction( mActions[LookupImplementation] );
+    menu->addAction( mActions[LookupReferencesForCursor] );
+    menu->addAction( mActions[LookupReferences] );
+
+    menuBar->addMenu(menu);
+
+    menu = new QMenu(tr("Se&rver"), this);
```

Comment:
S is already the shortcut for the Session menu, so I had to pick something else.